### PR TITLE
[PLATFORM-657] Core nav updates

### PR DESCRIPTION
**Pull request by mondoreale**

### CODE REVIEW

**Comment by mondoreale** [comment]
Desktop core

Screenshot 2019-05-22 12 52 14

---

Desktop Docs

Screenshot 2019-05-22 12 52 20

---

Desktop profile dropdown

Screenshot 2019-05-22 12 52 45

---

Tablet Core

Screenshot 2019-05-22 12 52 59

---

Table Docs

Screenshot 2019-05-22 12 53 03

---

Tablet profile dropdown

Screenshot 2019-05-22 12 53 09

**Comment by mondoreale** [comment]
@mattatgit I also fixed the height of the actual top nav on Tablet. Missed it and it was 80px instead of 64px. Same with the logo. Smaller on tablest.

File: app/src/shared/components/Nav/index.jsx [hunk 1]
```diff
@@ -41,59 +40,59 @@ const Nav = compose(
...
+            <DropdownItem
+                align="left"
+                label={I18n.t('general.docs')}
+                to={routes.docsIntroduction()}
+            >
+                <Link to={routes.docsIntroduction()}>
+                    <Translate value="general.introduction" />
+                </Link>
+                <Link to={routes.docsTutorials()}>
+                    <Translate value="general.getStarted" />
```
Comment (fonty1) [inline app/src/shared/components/Nav/index.jsx]: Typo here - should go to a `getting-started` route

File: app/src/userpages/components/DocsShortcuts/index.jsx [hunk 2]
```diff
@@ -3,19 +3,36 @@
...
-        {Object.keys(navigationLinks).map((key) => (
-            <Link key={key} to={navigationLinks[key]} target="_blank" rel="noopener noreferrer">
-                {key}
-            </Link>
-        ))}
+        <Link to={routes.docsIntroduction()} target="_blank" rel="noopener noreferrer">
+            <Translate value="general.introduction" />
+        </Link>
+        <Link to={routes.docsTutorials()} target="_blank" rel="noopener noreferrer">
+            <Translate value="general.getStarted" />
```
Comment (fonty1) [inline app/src/userpages/components/DocsShortcuts/index.jsx]: And here -> `get-started`

**Comment by fonty1** [comment]
So there isn't actually a `getting-started` route because the base `docs` route goes there, but that is wrong, see this ticket:
https://streamr.atlassian.net/browse/PLATFORM-696

So if you could add a `getting-started` route and make the base `docs` route lead to the `introduction` to close off that ticket it would be appreciated!

`<Route exact path={docs.main} component={GettingStartedPage} key="GettingStartedPage" />`
--->
`<Route exact path={docs.main} component={IntroductionPage} key="IntroductionPage" />`
+ 
`<Route exact path={docs.gettingStarted} component={GettingStartedPage} key="GettingStartedPage2" />`

**Comment by mondoreale** [comment]
Good notes guys. 👍  I'll address all of it before merging.

**Comment by mondoreale** [comment]
@fonty1 fixed [`PLATFORM-696`](https://streamr.atlassian.net/browse/PLATFORM-696) in 16c53e0. Made it so that `/docs` redirects to `/docs/introduction` – feels better than having introduction on both `/docs` and `/docs/introduction`.

💃

**Comment by fonty1** [comment]
> @fonty1 fixed [`PLATFORM-696`](https://streamr.atlassian.net/browse/PLATFORM-696) in [16c53e0](https://github.com/streamr-dev/streamr-platform/commit/16c53e0904e2569c12320200b9ab1b9e6fd299a1). Made it so that `/docs` redirects to `/docs/introduction` – feels better than having introduction on both `/docs` and `/docs/introduction`.
> 
> 💃

Yep, much better 👍

**Pull request merged**